release_notes/topics/14_0_0.adoc

```diff
@@ -0,0 +1,8 @@
+= Highlights
+
+== Client Policies and FAPI Support
+
+The {project_name} server has now official support for client policies and FAPI. This was preview in the previous versions, but now
```

Comment:
Change " FAPI. This was preview in the previous versions,"
TO
 Financial API (FAPI). This capability was previewed in earlier versions,

Reply:
@andymunro @tnorimat Thanks! I am changing " FAPI. This was preview in the previous versions," to " Financial-grade API (FAPI). This capability was previewed in earlier versions,"